kernel/src/capabilities.rs

```diff
@@ -72,3 +72,9 @@ pub unsafe trait UdpDriverCapability {}
 /// so this capability should not be distributed to capsules at all, as the port table should only be
 /// instantiated once by the kernel
 pub unsafe trait CreatePortTableCapability {}
+
+pub unsafe trait UdpVisCap {}
+
+pub unsafe trait IpVisCap {}
+
+pub unsafe trait NetCapCreateCap {}
```

Comment:
If we can have one capability here, that is well documented, then I'm happy to continue the policy of aggressively merging in the net folder.

In any case we want to avoid using shorthand, and instead use descriptive names like the other capabilities.